### PR TITLE
Add workshops tab

### PR DESCRIPTION
**Pull request by rgrandl**
[workshops.webm](https://github.com/ServiceWeaver/weaver/assets/1679018/b50720e2-0829-40b2-8e5a-ee01725ce0d2)

### CODE REVIEW

File: website/assets/css/common.css [hunk 1]
```diff
@@ -97,6 +97,14 @@ header .news:hover:not(.active) {
   color: red;
   border-bottom: 2pt solid gray;
 }
+header .workshops {
```
Comment (mwhittaker) [inline website/assets/css/common.css:100]: The "News" tab is red and the "Workshops" tab is purple. I think I prefer having all links be black or at most one link being a different color? What do you think?

File: website/assets/css/workshops.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+.workshops-content {
+  margin-left: 1rem;
+  padding-bottom: 1rem;
+}
+
+.workshops-date {
+  color: gray;
+  padding-bottom: 0.25rem;
+}
```
Comment (mwhittaker) [inline website/assets/css/workshops.css:39]: I think all the CSS above is unused?

File: website/assets/css/workshops.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+.workshops-table {
+  width: 70%;
```
Comment (mwhittaker) [inline website/assets/css/workshops.css:42]: On mobile, the 70% width looks a bit cramped. I recommend either changing this to 100%, or adding the following to change the width to 100% on small screens: 

```cs
@media (max-width: 800px) {
  .workshops-table {
    width: 100%;
  }
}
```

File: website/workshops.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        link: '<a href="https://gdg.community.dev/events/details/google-gdg-zurich-presents-service-weaver-workshop/">Register</a>',
+      },
+    ];
+
+    const workshops = document.getElementById('workshops');
+    const block = document.createElement('div');
+    block.classList.add('news-block');
+    block.innerHTML = `
+      <div class="news-content">
+        <div style="color:darkblue"><p>Please reach out to us if you're interested in hosting a Service Weaver workshop in your city or institution!</p></div>
```
Comment (mwhittaker) [inline website/workshops.html:39]: Can this text be black?

Comment (mwhittaker) [inline website/workshops.html:39]: Maybe add a sentence like "We're hosting a number of workshops where you can get hands-on experience writing Service Weaver apps. Use the links below to sign up."?

File: website/workshops.html
Comment (mwhittaker) [inline website/workshops.html:1]: You can simplify things a bit by hardcoding more html and doing less in the javascript: 

```html
<p style="color: darkblue;">
  Please reach out to us if you're interested in hosting a Service Weaver
  workshop in your city or institution!
<p>

<table id="workshops" class="workshops-table"></table>

<script>
  document.addEventListener('DOMContentLoaded', () => {
    const items = [
      {
        date: new Date('2023-06-27'),
        location: 'Berlin',
        link: '<a href="https://gdg.community.dev/events/details/google-gdg-berlin-golang-presents-service-weaver-workshop/">Register</a>',
      },
      {
        date: new Date('2023-06-30'),
        location: 'Zurich',
        link: '<a href="https://gdg.community.dev/events/details/google-gdg-zurich-presents-service-weaver-workshop/">Register</a>',
      },
    ];

    const table = document.getElementById('workshops');

    // Create table header row.
    const headerRow = document.createElement("tr")
    table.appendChild(headerRow)
    for (const key in items[0]) {
      const headerCell = document.createElement("th");
      headerCell.textContent = key.charAt(0).toUpperCase() + key.slice(1);
      headerRow.appendChild(headerCell);
    }

    // Create table data rows.
    const options = { year: 'numeric', month: 'long', day: 'numeric' };
    for (const item of items) {
      const itemRow = document.createElement("tr");
      table.appendChild(itemRow)

      const itemCellDate = document.createElement("td")
      itemCellDate.textContent = item.date.toLocaleDateString(undefined, options)
      itemRow.appendChild(itemCellDate)

      const itemCellLoc = document.createElement("td")
      itemCellLoc.textContent = item.location
      itemRow.appendChild(itemCellLoc)

      const itemCellLink = document.createElement("td")
      itemCellLink.innerHTML = item.link
      itemRow.appendChild(itemCellLink)
    }
  });
</script>
```

**Review by mwhittaker — APPROVED** [review]
Awesome! Thanks Robert. Looking forward to the workshop :)

File: website/assets/css/common.css (hunk 1 above)
Comment (rgrandl) [reply website/assets/css/common.css:100]: I don't have strong opinions. Changed to black.

File: website/workshops.html
Comment (rgrandl) [reply website/workshops.html:1]: Looks better. Thanks Michael

File: website/assets/css/workshops.css (hunk 3 above)
Comment (rgrandl) [reply website/assets/css/workshops.css]: Changed to 100% width. Thanks for checking on mobile.

File: website/workshops.html (hunk 4 above)
Comment (rgrandl) [reply website/workshops.html]: Good idea. Added.

Comment (rgrandl) [reply website/workshops.html]: it can be black :)

**Comment by rgrandl** [comment]
> Awesome! Thanks Robert. Looking forward to the workshops :)

Thanks Michael for the great review.

**Pull request merged**